test/functional/unlocker-e2e-specs.js

```diff
@@ -0,0 +1,53 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import _ from 'lodash';
+import AndroidDriver from '../..';
+import DEFAULT_CAPS from './desired';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+let defaultCaps = _.defaults({
+  androidInstallTimeout: 90000
+}, DEFAULT_CAPS);
+
+const AVD = "ANDROID_API_19";
+
+/*
+------------------------------------------------------------------------------
+TODO - In order to run the unlock2 e2e tests we need to create specific
+emulators avds each one of them setted with the current unlock type to test
+------------------------------------------------------------------------------
+*/
+describe('unlock tests', function () {
+  let driver;
+  before(() => {
+    driver = new AndroidDriver();
+  });
+  afterEach(async () => {
+    await driver.deleteSession();
+  });
+  /*
+  it('should unlock the device using a PIN', async () => {
```

Comment:
I'd prefer these to be skipped (https://mochajs.org/#inclusive-tests) rather than commented out. That way we can see that they are skipped in the test report, and audit what is there but not being run with a simple search.